lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -52,8 +52,20 @@ module.exports.prototype = {
                 return;
             }
 
+            var prevToken = file.getPrevToken(comment, {
+                includeComments: true
+            });
+
+            if (prevToken.type === 'Line') {
+                return;
+            }
+
+            if (prevToken.type === 'Punctuator' && prevToken.value === '{') {
```

Comment:
How do we decide on the default behavior here? 

It seems like if this rule is used 99% of the time with `allExcept: 'firstInBlock'`then that should be the default. But it would be speculation on both of our sides to try to guess which would be the 'more common' case.